docs-guides/source/stateful-models.md

```diff
@@ -227,6 +227,217 @@ potential runtime performance improvements.
 For instance, please check out the [2024 WWDC session](https://developer.apple.com/videos/play/wwdc2024/10159/) for an 
 example that uses the [Mistral 7B model](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.2)
 and utilizes the stateful prediction feature for improved performance on a GPU on a macbook pro.  
+The code for converting and deploying Mistral 7B model is released at https://huggingface.co/blog/mistral-coreml.
```

Comment:
How about link to https://github.com/huggingface/swift-transformers/tree/21b015691624ad103470370f0f255eb660579511/Examples/Mistral7B somewhere (save 1 indirection from above blog post)

Reply:
Good idea. Changed.